test/UnitTest/v1/test_casesNGSIv1.py

```diff
@@ -16,704 +16,974 @@
+        headers={'Content-Type':'appliction/json','fiware-service':'Abc','fiware-servicepath':'pqr'}
+        r=requests.post(url,data=json.dumps(data_ngsi10.subdata53),headers=headers)
+        print(r.content)
+
+        #get the created entity
+        url=brokerIp+"/ngsi10/entity/Result053"
+        r=requests.get(url)
+        print(r.content)
+
+        #passing DELETE in update payload
+        url=brokerIp+"/ngsi10/updateContext"
+        headers={'Content-Type':'appliction/json','fiware-service':'Abc','fiware-servicepath':'pqr'}
+        r=requests.post(url,data=json.dumps(data_ngsi10.subdata54),headers=headers)
+        print(r.content)
+
+        #get the created entity
+        url=brokerIp+"/ngsi10/entity/Result053"
+        r=requests.get(url)
+        print(r.content)
+        assert r.status_code == 404
+
+#testCase 34
+'''
+  To test the notifyContext request 
+'''
+def test_case34():
+	url=brokerIp+"/ngsi10/notifyContext"
+	headers={'Content-Type':'appliction/json'}
+	r=requests.post(url,data=json.dumps(data_ngsi10.subdata55),headers=headers)
+	print(r.content)
+	assert r.status_code == 200
+
+#testCase 35
+'''
+  To test unsubscribing feature
+'''
+def test_case35():
+	#create subscription
+	url=brokerIp+"/ngsi10/subscribeContext"
+        headers= {'Content-Type': 'application/json'}
+        r=requests.post(url,data=json.dumps(data_ngsi10.subdata56 ),headers=headers)
+        resp_content=r.content
+        resInJson= resp_content.decode('utf8').replace("'", '"')
+        resp=json.loads(resInJson)
+        print(resp)
+        resp=resp['subscribeResponse']
+        sid=resp['subscriptionId']
+        print(sid)
+
+	#unsubscribe Context
+	url=brokerIp+"/ngsi10/unsubscribeContext"
+	headers={'Content-Type': 'application/json'}
+	r=requests.post(url,json={"subscriptionId":sid,"originator":"POMN"},headers=headers)
+	print(r.content)
+	assert r.status_code == 200 
+
+#testCase 36
+'''
+  To test entity creation using other route
+'''
+def test_case36():
+        url=brokerIp+"/v1/updateContext"
+        headers={'Content-Type':'appliction/json'}
+        r=requests.post(url,data=json.dumps(data_ngsi10.subdata56),headers=headers)
+        print(r.content)
+	assert r.status_code == 200
+
+#testCase 37
+'''
+  To test and fetch unique entity
+'''
+def test_case37():
+	url=brokerIp+"/ngsi10/entity/Result14"
+	r=requests.get(url)
+	print(r.content)
+	assert r.status_code == 200
+
+#testCase 38
+'''
+  To test and fetch attribute specific to an entity
+'''
+def test_case38():
+	url=brokerIp+"/ngsi10/entity/Result14/pressure"
+	r=requests.get(url)
+	resp_content=r.content
+        resInJson= resp_content.decode('utf8').replace("'", '"')
+        resp=json.loads(resInJson)
+	print(r.content)
+        name=resp['name']
+        type1=resp['type']
+	val=resp['value']
+	if name=='pressure' and type1=='float' and val==55:
+		print("Validated")
+	else:
+		print("Not Validated")
+	assert r.status_code == 200
+
+
```

Comment:
Also test the remote broker case, where entity lies on another broker and subscription is done on some other broker.